media/js/base/mozilla-notification-banner-init.js

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(function() {
```

Comment:
Yeah, this is not very obvious but we need the notification GA event to fire after [this event](https://github.com/mozilla/bedrock/blob/master/media/js/base/core-datalayer-init.js#L26-L28). I'll add a comment here to make this more obvious as to why.